packages/boa/requirements.txt

```diff
@@ -1,2 +1,3 @@
 numpy==1.17.4
-
+jieba==0.42.1
+sklearn==0.0
```

Comment:
I'm not sure if we should do this here :(

Reply:
If not, it will throw error: 'not fount module 'jieba'' although I have 'jieba' use pip in my local.
And it means 'jieba' must install into dir: '.miniconda'?

Reply:
Yes, i think we can just keep jieba and sklearn here in requirements. After we have completed plugin runtime, this can be removed in the future.

Reply:
Thanks and LGTM :)

Reply:
Resolved.